app/controllers/application_controller.rb

```diff
@@ -3,7 +3,7 @@ class ApplicationController < ActionController::Base
 
   protect_from_forgery with: :null_session
   before_action :configure_permitted_parameters, if: :devise_controller?
-
+  skip_before_action :verify_authenticity_token #todo
```

Comment:
This is OK, but add a comment explaining why you had to do this above. Remember, todo is always in caps and with a description:

```rb
# TODO, review why token is not working for an API.
skip_before_action :verify_authenticity_token
```

Reply:
done!